code/game/objects/items/cards_ids.dm

```diff
@@ -686,6 +686,112 @@ update_label("John Doe", "Clowny")
 	access = J.get_access()
 	. = ..()
 
+// Temporary IDs
+/obj/item/card/id/temp
+	registered_name = "Temporary ID"
+	lefthand_file = 'icons/mob/inhands/equipment/idcards_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/idcards_righthand.dmi'
+
+/obj/item/card/id/temp/lost
+	registered_name = "Lost ID"
+	assignment = "Unknown"
+	desc = "A card used to provide ID and determine access across the station. It has a bunch of scribbled numbers on the side of it."
+
+/obj/item/card/id/temp/lost/Initialize()
+	var/list/accesses = list(
+			"[ACCESS_CARGO]" = 12,
+			"[ACCESS_COURT]" = 6,
+			"[ACCESS_MEDICAL]" = 10,
+			"[ACCESS_GENETICS]" = 10,
+			"[ACCESS_MORGUE]" = 7,
+			"[ACCESS_TOX]" = 10,
+			"[ACCESS_TOX_STORAGE]" = 8,
+			"[ACCESS_CHEMISTRY]" = 10,
+			"[ACCESS_BRIGPHYS]" = 10,
+			"[ACCESS_BAR]" = 8,
+			"[ACCESS_JANITOR]" = 8,
+			"[ACCESS_ENGINE]" = 10,
+			"[ACCESS_ENGINE_EQUIP]" = 10,
+			"[ACCESS_MAINT_TUNNELS]" = 20,
+			"[ACCESS_CHAPEL_OFFICE]" = 7,
+			"[ACCESS_TECH_STORAGE]" = 10,
+			"[ACCESS_ATMOSPHERICS]" = 10,
+			"[ACCESS_CONSTRUCTION]" = 12,
+			"[ACCESS_KITCHEN]" = 8,
+			"[ACCESS_HYDROPONICS]" = 10,
+			"[ACCESS_LIBRARY]" = 6,
+			"[ACCESS_LAWYER]" = 7,
+			"[ACCESS_ROBOTICS]" = 10,
+			"[ACCESS_EXPLORATION]" = 8,
+			"[ACCESS_SURGERY]" = 10,
+			"[ACCESS_THEATRE]" = 8,
+			"[ACCESS_RESEARCH]" = 12,
+			"[ACCESS_MINING]" = 11,
+			"[ACCESS_MAILSORTING]" = 9,
+			"[ACCESS_MINING_STATION]" = 10,
+			"[ACCESS_XENOBIOLOGY]" = 15,
+			"[ACCESS_RC_ANNOUNCE]" = 10,
+			"[ACCESS_CLONING]" = 5,
+			"[ACCESS_MECH_MINING]" = 5,
+			"[ACCESS_MECH_MEDICAL]" = 7,
+			"[ACCESS_MECH_SCIENCE]" = 7,
+			"[ACCESS_MECH_ENGINE]" = 7,
+			"[ACCESS_AUX_BASE]" = 10,
+		)
+	access = get_random_access(accesses, 12)
+	. = ..()
```

Comment:
```suggestion
#define TEXTIFY_ACCESS_CODE(code) #code
/obj/item/card/id/temp/lost
	registered_name = "Lost ID"
	assignment = "Unknown"
	desc = "A card used to provide ID and determine access across the station. It has a bunch of scribbled numbers on the side of it."
	var/static/list/random_access_list = list(
			TEXTIFY_ACCESS_CODE(ACCESS_CARGO) = 12,
			TEXTIFY_ACCESS_CODE(ACCESS_COURT) = 6,
			TEXTIFY_ACCESS_CODE(ACCESS_MEDICAL) = 10,
			TEXTIFY_ACCESS_CODE(ACCESS_GENETICS) = 10,
			TEXTIFY_ACCESS_CODE(ACCESS_MORGUE) = 7,
			TEXTIFY_ACCESS_CODE(ACCESS_TOX) = 10,
			TEXTIFY_ACCESS_CODE(ACCESS_TOX_STORAGE) = 8,
			TEXTIFY_ACCESS_CODE(ACCESS_CHEMISTRY) = 10,
			TEXTIFY_ACCESS_CODE(ACCESS_BRIGPHYS) = 10,
			TEXTIFY_ACCESS_CODE(ACCESS_BAR) = 8,
			TEXTIFY_ACCESS_CODE(ACCESS_JANITOR) = 8,
			TEXTIFY_ACCESS_CODE(ACCESS_ENGINE) = 10,
			TEXTIFY_ACCESS_CODE(ACCESS_ENGINE_EQUIP) = 10,
			TEXTIFY_ACCESS_CODE(ACCESS_MAINT_TUNNELS) = 20,
			TEXTIFY_ACCESS_CODE(ACCESS_CHAPEL_OFFICE) = 7,
			TEXTIFY_ACCESS_CODE(ACCESS_TECH_STORAGE) = 10,
			TEXTIFY_ACCESS_CODE(ACCESS_ATMOSPHERICS) = 10,
			TEXTIFY_ACCESS_CODE(ACCESS_CONSTRUCTION) = 12,
			TEXTIFY_ACCESS_CODE(ACCESS_KITCHEN) = 8,
			TEXTIFY_ACCESS_CODE(ACCESS_HYDROPONICS) = 10,
			TEXTIFY_ACCESS_CODE(ACCESS_LIBRARY) = 6,
			TEXTIFY_ACCESS_CODE(ACCESS_LAWYER) = 7,
			TEXTIFY_ACCESS_CODE(ACCESS_ROBOTICS) = 10,
			TEXTIFY_ACCESS_CODE(ACCESS_EXPLORATION) = 8,
			TEXTIFY_ACCESS_CODE(ACCESS_SURGERY) = 10,
			TEXTIFY_ACCESS_CODE(ACCESS_THEATRE) = 8,
			TEXTIFY_ACCESS_CODE(ACCESS_RESEARCH) = 12,
			TEXTIFY_ACCESS_CODE(ACCESS_MINING) = 11,
			TEXTIFY_ACCESS_CODE(ACCESS_MAILSORTING) = 9,
			TEXTIFY_ACCESS_CODE(ACCESS_MINING_STATION) = 10,
			TEXTIFY_ACCESS_CODE(ACCESS_XENOBIOLOGY) = 15,
			TEXTIFY_ACCESS_CODE(ACCESS_RC_ANNOUNCE) = 10,
			TEXTIFY_ACCESS_CODE(ACCESS_CLONING) = 5,
			TEXTIFY_ACCESS_CODE(ACCESS_MECH_MINING) = 5,
			TEXTIFY_ACCESS_CODE(ACCESS_MECH_MEDICAL) = 7,
			TEXTIFY_ACCESS_CODE(ACCESS_MECH_SCIENCE) = 7,
			TEXTIFY_ACCESS_CODE(ACCESS_MECH_ENGINE) = 7,
			TEXTIFY_ACCESS_CODE(ACCESS_AUX_BASE) = 10,
		)
#undef TEXTIFY_ACCESS_CODE

/obj/item/card/id/temp/lost/Initialize()
	access = get_random_access(random_access_list, 12)
	. = ..()
```

Reply:
The suggested code here just won't work since its not textifying anything and it doesn't make sense to do it like this compared to just doing regular string interpolation